### DISCUSSION TITLE
Why do packages get discovered after couple of days of their release?

### DISCUSSION THREAD

**Discussion by reddyshyam** [Devolutions/UniGetUI#1773] [Q&A] [answered]
At the moment, I see KeePass 2.56 released on 4th Feb, today it is already 7th. Source is winget.

**Comment by Spacellary** [comment]
Yeah, definitely a Winget'ey situation, as Scoop has the latest for example.

**Reply by reddyshyam** [reply]
Hi @Spacellary,

Thanks for your reply. Sorry, I am just a noob, which has a better repository winget, scoop or chocolatey? Or each has it's own favouritism?

Also can we tell winget that a package has an update or it has its own process?

**Reply by Spacellary** [reply]
@reddyshyam I am definitely biased towards Scoop. I much prefer its focus on Portable installations and how easy it is to roll back and forth between versions of you're willing to use the terminal command `scoop reset app@version`.

I'm not sure about signaling Winget about updates, I think their process might just be slower, you can cheat their Winget -cli repo and see if you find anything useful or any #issues about it.

**Comment by reddyshyam** [comment]
> @reddyshyam I am definitely biased towards Scoop. I much prefer its focus on Portable installations and how easy it is to roll back and forth between versions of you're willing to use the terminal command `scoop reset app@version`.
> 
> I'm not sure about signaling Winget about updates, I think their process might just be slower, you can cheat their Winget -cli repo and see if you find anything useful or any #issues about it.

Thanks @Spacellary. Since I am not familiar with either of the repositories, I will keep wishing and praying someone who maintains, wakes up and release them. 😀

**Comment by Spacellary** [comment, marked as answer]
@reddyshyam I think I see the issue, so 2 days ago it was already submitted, but the Pull Request is still open and the merge was blocked, it seems they are discussing to add it here:

https://github.com/microsoft/winget-pkgs/pull/137901

If you want to understand what happened I recommend checking this discussion.

I will recommend using Scoop whenever possible as I find these issues don't happen often there.

**Reply by reddyshyam** [reply]
Thanks again @Spacellary. Man! That discussion is beyond my scope. 😀I have added scoop repo but it didnt pick up keepass either. Should the source be scoop for it to pick any updates?

**Reply by Spacellary** [reply]
@reddyshyam Just uninstall yours (make backups if needed) and install from the Scoop source. You'll get updates as usual from that source aswell.

**Reply by reddyshyam** [reply]
Thanks @Spacellary. This morning got the update from winget but will surely explore that option. Thanks for your time.  🙏 Will close this discussion now.
